HelpSource/Classes/ServerOptions.schelp

```diff
@@ -122,8 +122,10 @@ method:: restrictedPath
 Allows you to restrict the system paths in which the server is allowed to read/write files during running. A nil value (the default) means no restriction. Otherwise, set it as a string representing a single path.
 
 method:: threads
-Number of audio threads that are spawned by supernova. For scsynth this value is ignored. If it is code::nil::or 0, it
-uses the one thread per CPU. Default is code::nil::.
+Number of audio threads that are spawned by supernova. For scsynth this value is ignored. If it is code::nil::or 0, it uses one thread CPU. Default is code::nil::.
```

Comment:
suggestion:
`it uses one thread CPU` -> `it uses one thread per CPU core`

Reply:
ouch, thanks!